scripts/tests/relay-graceful-shutdown.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "./scripts/tests/lib.sh"
+
+# Arrange: Setup a relayed connection
+install_iptables_drop_rules
+client_curl_resource "172.20.0.100/get"
```

Comment:
I think it might be better to use the `dns.httpbin` resource? It is a bit more obvious that it's a resource

Reply:
We use the IP a lot in other tests too, I just copied that.